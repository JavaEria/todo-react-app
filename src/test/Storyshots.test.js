jest.mock("react-dom", () => {
    return {
        render: () => null,
        unmountComponentAtNode: () => null,
        findDOMNode: () => { return {} },
    };
 });

import initStoryshots from '@storybook/addon-storyshots';

initStoryshots();