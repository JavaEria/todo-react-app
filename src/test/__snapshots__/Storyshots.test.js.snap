// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots TodoItem Single Todo Item 1`] = `
<div
  className="TodoItem"
>
  <input
    checked=""
    className="checkmark"
    onChange={[Function]}
    type="checkbox"
  />
  <span
    className="text"
  >
    <p
      className="label"
      onClick={[Function]}
    >
      Todo1
    </p>
  </span>
  <button
    className="cross"
    onClick={[Function]}
  >
    X
  </button>
</div>
`;
